src/core/Pricing/PricingCards.tsx

```diff
@@ -0,0 +1,163 @@
+import React, { Fragment } from "react";
+import type { PricingDataFeature } from "./types";
+import { determineThemeColor } from "../styles/colors/utils";
+import { ColorClass, Theme } from "../styles/colors/types";
+import Icon from "../Icon";
+import FeaturedLink from "../FeaturedLink";
+import { IconName } from "../Icon/types";
+
+type PricingCardsProps = {
+  data: PricingDataFeature[];
+  theme?: Theme;
```

Comment:
In time, this will be set further up in a context and accessed within the component instead of having to drill it in here